types/three/three-core.d.ts

```diff
@@ -6065,6 +6051,26 @@ export class WebGLLights {
     get(light: any): any;
 }
 
+/** 
+ * An object with a series of statistical information about the graphics board memory and the rendering process. 
+ */ 
+export class WebGLInfo { 
+    autoReset: boolean; 
+    memory: { 
+        geometries: number; 
+        textures: number; 
+    }; 
+    programs: number | null; 
+    render: { 
```

Comment:
render appears to have a `frame` member. https://github.com/mrdoob/three.js/blob/dev/src/renderers/webgl/WebGLInfo.js#L13